### PR TITLE
feat(frontend): pageinate reports on timed-admin/detail

### LINKED ISSUE #114: Pagination of timed-admin/detail view
implement pagination for orders and reports on timed-admin/detail view

### PR DESCRIPTION
**Pull request by Yelinz**
fix #114

### CODE REVIEW

File: frontend/app/locales/de/translations.js [hunk 1]
```diff
@@ -160,7 +160,10 @@ export default {
       employee: 'Mitarbeiter',
       description: 'Beschreibung',
       acknowledged: 'Bestätigt',
-      amount: 'Anzahl Stunden'
+      amount: 'Anzahl Stunden',
+      errorLoading:
+        'Es ist ein Problem aufgetreten beim holen der Daten. Bitte versuchen Sie es erneut',
```
Comment (velrest) [inline frontend/app/locales/de/translations.js]:
```suggestion
        'Es ist ein Problem aufgetreten beim laden der Daten. Bitte versuchen Sie es erneut',
```

File: frontend/app/timed-admin/detail/controller.js [hunk 2]
```diff
@@ -16,7 +16,13 @@ export default Controller.extend({
   loading: alias('fetchModels.isRunning'),
   project: alias('fetchModels.lastSuccessful.value.project'),
   orders: alias('fetchModels.lastSuccessful.value.orders'),
-  reports: alias('fetchModels.lastSuccessful.value.reports'),
+  reportsPage: 1,
+  reportsNext: false,
+
+  init() {
+    this._super(...arguments)
+    this.set('reports', [])
```
Comment (velrest) [inline frontend/app/timed-admin/detail/controller.js]: Reports needs to be reset every time the route model changes. I think this way you have state which consists even if you navigate to another project.

Comment (velrest) [reply frontend/app/timed-admin/detail/controller.js]: Same with `reportsPage` and `reportsNext`

File: frontend/app/timed-admin/detail/controller.js [hunk 3]
```diff
@@ -80,10 +82,30 @@ export default Controller.extend({
...
+        project: this.get('model'),
+        include: 'user',
+        ordering: '-date',
+        page: this.get('reportsPage'),
+        page_size: 20
+      })
+
+      reports.forEach(report => this.reports.pushObject(report))
+      this.set('reportsPage', this.get('reportsPage') + 1)
+      this.set('reportsNext', !!reports.get('links.next'))
```
Comment (velrest) [inline frontend/app/timed-admin/detail/controller.js]: `Boolean(reports.get('links.next'))` would probably be a more readable way

File: frontend/app/timed-admin/detail/controller.js (hunk 2 above)
Comment (Yelinz) [reply frontend/app/timed-admin/detail/controller.js]: Moved to router controllerSetup hook

File: frontend/app/timed-admin/detail/controller.js (hunk 3 above)
Comment (Yelinz) [reply frontend/app/timed-admin/detail/controller.js]: Changed

**Pull request merged**